test_data_studies/commands.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash -xe
+
+export PGDATESTYLE="ISO, MDY"
+
+studies=`ls test_data_studies/studies`
+
+echo "DROPPING ENV... "
+qiita_env drop --no-ask-for-confirmation
+echo "Ok"
+
+echo "MAKING ENV... "
+qiita_env make --add-demo-user --no-load-ontologies
+echo "Ok"
+
+# Inserting all the information for each study
+echo "INSERTING STUDIES"
+mkdir -p temp
+base_id=0
+for i in ${studies[@]}; do
```

Comment:
Something like that. I was thinking in making copies of all the files that get moved and then put them back in place. Similar to the otu table.